pub_sub/go/sdk/order-processor/app.go

```diff
@@ -19,10 +19,12 @@ var sub = &common.Subscription{
 }
 
 func main() {
-	//read app-port passed thru command line
+	// read app-port passed thru 'dapr run' command line
```

Comment:
```suggestion
	// read app-port passed through 'dapr run' command line
```
Comments to end with period.